crates/vimfuncs/build.rs

```diff
@@ -26,160 +28,158 @@ pub struct IfDef {
     pub proc_else: Box<FuncToken>,
 }
 
-impl FuncToken {
-    fn string(self) -> String {
-        match self {
-            FuncToken::String(s) => s,
-            _ => unreachable!("{:?}", self),
-        }
-    }
-
-    fn number(self) -> usize {
-        match self {
-            FuncToken::Number(s) => s,
-            _ => unreachable!("{:?}", self),
-        }
-    }
-
-    fn identifier(self) -> String {
```

Comment:
i don't get why you removed these

Reply:
because these functions used on incorrect types (e.g. number on FuncToken::String) will throw an error, and it's not really unreachable 

I replaced them with pattern matching the enum because I think it's more idiomatic. 

Let me know what you think. I can revert back if needed